rust/client-tunnel/Cargo.toml

```diff
@@ -6,6 +6,21 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+anyhow = { version = "1.0" }
+# Will be needed to safely revert `/etc/resolv.conf`
+atomicwrites = "0.4.3"
```

Comment:
Is it worth throwing this into `[target.'cfg(target_os = "linux")'.dependencies]`?

Reply:
Oh I think it's used on Windows for the device ID too. I'll clarify that or reduce it to Linux before merging 👍